training/src/anemoi/training/config/hardware/example.yaml

```diff
@@ -1,10 +1,5 @@
 ---
 defaults:
-- paths: example
-- files: example
-
-# number of GPUs per node and number of nodes (for DDP)
-accelerator: auto
-num_gpus_per_node: 1
-num_nodes: 1
-num_gpus_per_model: 1
+  - storage: example
```

Comment:
If I am following correctly, shouldn't this be system/example.yaml? (I think that's why now the unit tests are failing too)